Azure/azure-cli
# [Search] `az search service create`: Add `--hosting-mode` argument to support S3HD SKU

**Related command**

search

**Description**

Add support for the S3HD sku to az cli
https://docs.microsoft.com/en-us/azure/search/search-sku-tier#tier-descriptions
https://docs.microsoft.com/en-us/rest/api/searchmanagement/2021-04-01-preview/services/create-or-update

**Testing Guide**

 az search service create -n {name} -g {rg} --sku standard3 --replica-count 1 --partition-count 1 --hosting-mode highDensity
Create an S3HD service with 1 replica and 1 partition

---

This checklist is used to make sure that common guidelines for a pull request are followed.
